### ISSUE TITLE
Multiple Thaumic Bases blocks incorrectly labeled as infusion stabilizers

### ISSUE THREAD

**Issue by budak7273** [GTNewHorizons/GT-New-Horizons-Modpack#15540]
### Your GTNH Discord Username

robb4

### Your Pack Version

2.5.1

### Your Server

Private Server

### Java Version

Java 17

### Type of Server

Vanilla Forge

### Your Expectation

Build with "ancient iron slabs" from Thaumic Bases and have it count as an infusion stabilizer according to the Arcane Abacus, since the tooltip says it is.

![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/25965766/8f5fb334-ccb3-4f52-985f-7c88e9889ba8)

### The Reality

The blocks don't actually offer any infusion stability.

![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/25965766/b76a4721-87fc-4a31-9777-c1acf9e62093)

Also doesn't work with double slabs.

![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/25965766/c5400c1d-b004-46e5-8f8e-0a57f7b77010)

Compare with this setup using valid infusion stabilizers:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/25965766/dda60c93-b727-4669-af16-ccd19cccf488)

### Your Proposal

Resolve the inaccurate tooltip by either making the following blocks actually stabilizers, or removing the tooltip.

The following blocks claim they are stabilizers in the tooltip, but aren't actually stabilizers according to the Arcane Abacus:
- `thaumicbases:genericSlab:4` (Ancient Diamond Slab)
- `thaumicbases:genericSlab:5` (Ancient Gold Slab)
- `thaumicbases:genericSlab:6` (Ancient Iron Slab)
- `thaumicbases:genericSlab_full:4` (Ancient Diamond Slab) [double slab]
- `thaumicbases:genericSlab_full:5` (Ancient Gold Slab) [double slab]
- `thaumicbases:genericSlab_full:6` (Ancient Iron Slab) [double slab]

Also, the following blocks _do not_ currently have the stabilizer tooltip, and do not count as stabilizers. I propose that their behavior should match their slab variants.
- `thaumicbases:oldIron` (Ancient Iron Block)
- `thaumicbases:oldGold` (Ancient Gold Block)
- `thaumicbases:oldDiamond` (Ancient Diamond Block)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
on the first proposal:
ancient iron/diamond/gold slabs dont look craftable to me.

for the second:
ancient iron/gold/diamond block are obtained by chiselling the normal ones. Which makes sense for purely cosmetic blocks, but means they should not be made into stabilizers.

**Comment by budak7273** [comment]
Oops, I wasn't aware they were uncraftable, I don't have all the thaum research unlocked in this world so I assumed it was gated by something I didn't have access to and/or there wasn't NEI support for the recipe. The questbook says in multiple places to not trust NEI recipes for thaum.

Is there a reason they're not craftable? [FTB wiki](https://ftbwiki.org/Ancient_Iron_Slab) claims their recipe would be a not-very-greg 3 blocks to six slabs crafting table recipe. I can see why that particular recipe was removed.

I should have clarified: at the end of the day I don't really care if they are stabilizers or not - I just don't want them to have the "I am a stabilizer" tool-tip if they aren't stabilizers, since I search "infusion stabilizer" to see my options (is there another way I should be using instead)? It's still a bug, and the bug is, "Multiple Thaumic Bases blocks incorrectly labeled as infusion stabilizers"

**Comment by budak7273** [comment]
Looking into this further has revealed more oddities.

Ancient Iron/Gold/Diamond Blocks can be chiseled as you mentioned, but you can also craft them in the Arcane Workbench with an entropy crystal block. An astute NEI-er would notice and never use the arcane workbench recipe.

The following blocks (and their slab variants) work as infusion stabilizers and have valid slab recipes in the Cutting Machine. They can't be chiseled, they only have the Arcane Workbench crafting recipe.
- Ancient Cobblestone
- Ancient Mossy Cobblestone

Ancient Lapis Lazuli Blocks and slabs work as stabilizers. Their slabs don't have a recipe but you can get one slab from the `L.E.S.U. Upgrade - EV Tier` quest.

It's not really clear to me if this family of blocks is intended to be decoration, stabilizer, or both. If iron/gold/diamond are intended as purely decorative, then why do the cobbles and lapis get a different treatment?

Seems like there are more decisions to make here. Should I split this into multiple issues with some as suggestions and some as bug reports?

**Comment by chochem** [comment]
that quest reward looks very out of place for sure.

**Comment by AdolMors** [comment]
I saw abacus has its own implementation of checking. I am more curious about if this logic is identical to Thaumcraft's stability checking so the abacus can be fully trusted.

**Comment by Graymare** [comment]
Agree that if said blocks are not valid infusion altar stabilizers, it would be preferable to remove the tooltip that says they may be usable.  Gold and Iron blocks are probably a little too easy to get with just Chisel, should at least have an Arcane Worktable requirement / Thaumcraft ingredient like Ancient Cobble does.

**Comment by budak7273** [comment]
As it stands, I think mob heads are the cheapest infusion stabilizer (just need a mob farm), followed by ancient cobblestone (1 entropy crystal cluster for 8).

**Comment by budak7273** [comment]
Pretty sure this is still an issue (commenting to prevent auto close)
